README.md

```diff
@@ -2,6 +2,10 @@
 
 This tool bisects either Rust nightlies or CI artifacts.
 
+## Setup
+
+Optionally, set the `RUST_SRC_REPO` environment variable to point at your local copy of https://github.com/rust-lang/rust. If the variable is not defined, `cargo-bisect-rustc` will clone it for you which takes a while.
```

Comment:
One detail missing here is that this has to be set at compile time. It's imported with the `option_env!` macro. See [here](https://doc.rust-lang.org/std/macro.option_env.html)